protocols/target_system_provider/target_system_provider.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package target_system_provider;
+
+// Service for providing target systems which can safely be connected to for execution of attacker commands.
+service TargetSystemProvider {
+    // Request to be provided a target system
+    // Returns error code UNAVAILABLE if no target system can currently be aqcuired. May be retried.
+    rpc AcquireTargetSystem (AcquisitionRequest) returns (AcquisitionResult) {}
```

Comment:
Something that completely went over my head when we talked about this was authentication. Is this request authenticated in any way or can anyone ask and acquire target systems? If we're lacking authentication it might be nice to create a new user story for it